pkg/scheduler/api/job_info.go

```diff
@@ -481,7 +481,17 @@ func (ji *JobInfo) GetMinResources() *Resource {
 		return EmptyResource()
 	}
 
-	return NewResource(*ji.PodGroup.Spec.MinResources)
+	resources := v1.ResourceList{}
+
+	for k, v := range *ji.PodGroup.Spec.MinResources {
```

Comment:
How about to prevent it from the source where min is calculated?
code is located at
https://github.com/volcano-sh/volcano/blob/015f86b9879d9204ab1607bd6e0f07189c7ecb5e/pkg/controllers/util/util.go#L12